src/gc/gcee.cpp

```diff
@@ -392,7 +392,7 @@ size_t GCHeap::GetNow()
 
 void ProfScanRootsHelper(Object** ppObject, ScanContext *pSC, DWORD dwFlags)
 {
-#if defined(GC_PROFILING) || defined(FEATURE_EVENT_TRACE)
```

Comment:
cc: @Maoni0 as there are a few changes here to enable tracing on Linux.  Can you please take a look?